packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.ts

```diff
@@ -352,7 +352,8 @@ export function useShopperBasketsMutation<Action extends ShopperBasketMutationTy
                 if (
                     action === 'createBasket' ||
                     action === 'transferBasket' ||
-                    action === 'mergeBasket'
+                    action === 'mergeBasket' ||
+                    action === 'addItemToBasket'
```

Comment:
Good catch!

Reply:
(If i understand this correctly, this PR will be closed after spike is done right? We need to make sure this is fixed in `develop`)

Reply:
@kevinxh yes, this is just a POC PR, it will not be merged after spike. I'd expect we have done this somewhere? or I can create a PR to add that